### DISCUSSION TITLE
Shortcut to minimize/unminimize to tray

### DISCUSSION THREAD

**Discussion by mattiklock** [TriliumNext/Trilium#2564] [Q&A]
I love the new tray functionality! Is it possible to bind a key to show and hide the Trilium window(s)?

I'm on Kubuntu. With Signal, Alt-F4 accomplishes the "hide" part; with Trilium it just kills the window entirely, or the app if the window is the last one left.

**Comment by zadam** [comment]
Hi, there isn't a keyboard shortcut for minimize to tray (yet).
